src/components/Amalgamation/BusinessTable.vue

```diff
@@ -181,7 +181,7 @@ export default class BusinessTable extends Mixins(AmalgamationMixin) {
   @Emit('valid')
   private emitValidity (): boolean {
     return (
-      (this.businesses.length > 0) &&
+      (this.businesses.length > 1) &&
```

Comment:
To be specific, the rule is "2 or more", so it might be better to use ">= 2".

Also, is there any way to have a unit test for this?